### PR TITLE
Adding documentation for the logout command

### PR DESCRIPTION
**Pull request by mkreibe**
As a followup to [Issue 660](https://github.com/concourse/concourse/issues/660) and [Pull request 114](https://github.com/concourse/fly/pull/114). This is the documentation for the `logout` command.

### CODE REVIEW

File: docs/using-concourse/fly-cli.any [hunk 1]
```diff
@@ -78,6 +78,27 @@ flag. Once you've learned what the commands do, you may want to consult
...
 }
 
+\section{\code{login}\aux{: Authenticating with and saving Concourse targets}}{fly-login}{
+
+  There are cases when you would like to remove all evidence of a particular
+  configuration. This is achieve by the \code{logout} command. There are two
+  variants of this command, one to get rid of a specific target, and another
+  to remove all targets from the \code{~/.flyrc} file.
+
+  To remove a specific target run:
```
Comment (vito) [inline docs/using-concourse/fly-cli.any]: Need a blank line after this for it to format properly.

File: docs/using-concourse/fly-cli.any [hunk 2]
```diff
@@ -78,6 +78,27 @@ flag. Once you've learned what the commands do, you may want to consult
...
+  \codeblock{bash}{
+  $ fly -t example logout
+  }
+
+  To remove all targets run:
```
Comment (vito) [inline docs/using-concourse/fly-cli.any]: (same here)

File: docs/using-concourse/fly-cli.any [hunk 3]
```diff
@@ -78,6 +78,27 @@ flag. Once you've learned what the commands do, you may want to consult
   \code{example} Concourse instance.
 }
 
+\section{\code{login}\aux{: Authenticating with and saving Concourse targets}}{fly-login}{
+
+  There are cases when you would like to remove all evidence of a particular
+  configuration. This is achieve by the \code{logout} command. There are two
```
Comment (vito) [inline docs/using-concourse/fly-cli.any]: should probably say 'target' - 'configuration' is a bit vague

also achieve -> achieved

File: docs/using-concourse/fly-cli.any [hunk 4]
```diff
@@ -78,6 +78,27 @@ flag. Once you've learned what the commands do, you may want to consult
   \code{example} Concourse instance.
 }
 
+\section{\code{login}\aux{: Authenticating with and saving Concourse targets}}{fly-login}{
+
```
Comment (vito) [inline docs/using-concourse/fly-cli.any]: kill blank line please

File: docs/using-concourse/fly-cli.any (hunk 3 above)
Comment (mkreibe) [inline docs/using-concourse/fly-cli.any]: Done and Done.

File: docs/using-concourse/fly-cli.any (hunk 1 above)
Comment (mkreibe) [inline docs/using-concourse/fly-cli.any]: Done.

File: docs/using-concourse/fly-cli.any (hunk 2 above)
Comment (mkreibe) [inline docs/using-concourse/fly-cli.any]: Done

**Comment by vito** [comment]
thanks!

**Pull request merged**